Packs/ServiceNow/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "ServiceNow",
     "description": "Use The ServiceNow IT Service Management (ITSM) solution to modernize the way you manage and deliver services to your users.",
     "support": "xsoar",
-    "currentVersion": "2.6.13",
```

This can be a minor chence 
```suggestion
    "currentVersion": "2.7.0",
```